bokeh/layouts.py

```diff
@@ -49,7 +51,11 @@
 # General API
 #-----------------------------------------------------------------------------
 
-def row(*args, **kwargs):
+
+def row(*args: LayoutDOM,
+        sizing_mode: Optional[LayoutDOM.sizing_mode] = None,
```

Comment:
`LayoutDOM.sizing_mode` should be replaced with `SizingMode` enum from `bokeh.core.enums`.

Reply:
Changed as discussed